Module not found: Error: Package path `./lib/uri/utils` is not exported from package.json

We have imported `import {getPath} from 'ts-closure-library/lib/uri/utils';` from the library iin our application but getting the bellow error during `ng build`

```
Error: Module not found: Error: Package path ./lib/uri/utils is not exported from package .../ui/node_modules/ts-closure-library (see exports field in .../ui/node_modules/ts-closure-library/package.json)
```

When we added ` "./lib/uri/utils": "./lib/uri/utils.js",` into the `exports` list of [package.json](https://github.com/DreierF/ts-closure-library/blob/master/package.json#L128), it seems to work fine.

Tried to create a PR but don't have write access. Can this change be added into the repo? 

Thanks, Dilip

--- a/package.json
+++ b/package.json
@@ -126,6 +126,7 @@
     "./lib/ui/tree/treenode": "./lib/ui/tree/treenode.js",
     "./lib/ui/zippy": "./lib/ui/zippy.js",
     "./lib/uri/uri": "./lib/uri/uri.js",
+    "./lib/uri/utils": "./lib/uri/utils.js",
     "./lib/useragent/useragent": "./lib/useragent/useragent.js"
   }
 }

